### ISSUE TITLE
"more" in SetProperties and DeleteProperties should be expandable

### ISSUE THREAD

**Issue by Stevenic** [microsoft/BotFramework-Composer#4282]
## Is your feature request related to a problem? Please describe.

It's fine that we limit the number of properties shown for `SetProperties` and `DeleteProperties` within the flow to 3 but the "2 more" subtitle should be a clickable link that expands to show all of your property assignments:

![image](https://user-images.githubusercontent.com/698939/94337500-a4eb7a00-ff9f-11ea-8fc9-4bc8a5c82796.png)

**Comment by Stevenic** [comment]
The property list is a bit hard to parse within the property editor so what I'd like is a more "at a glance" view of the full list of properties I'm manipulating.

**Comment by yeze322** [comment]
We can support that.

**Comment by a-b-r-o-w-n** [comment]
@sangwoohaan Can you take a look at this?

**Comment by Stevenic** [comment]
Any word on when this will get in?
